package.json

```diff
@@ -32,7 +32,7 @@
     "@typescript-eslint/parser": "4.28.0",
     "async-retry": "1.2.3",
     "buffer-replace": "1.0.0",
-    "cheerio": "1.0.0-rc.3",
+    "cheerio": "v1.0.0-rc.10",
```

Comment:
This also needs to update the `yarn.lock` file.

You can do that by running `yarn install`

Reply:
Done ☑️

Reply:
I think the `v` prefix should be removed too 

https://www.npmjs.com/package/cheerio

<img width="193" alt="image" src="https://user-images.githubusercontent.com/229881/158276388-e921e36a-b597-47be-9f9b-9b6c3b3ed2a8.png">

Reply:
I have removed it 👍🏻